enhancements/api-review/add-new-CRD-ImageSourceDigestPolicy-and-ImageSourceTagPolicy-to-config.openshift.io.md

```diff
@@ -0,0 +1,400 @@
+vertex, adding an unidirectional edge between adjacent mirrors from the same mirror list. If the graph is cyclic graph, the topological sort will start
+from the first vertex after the alphabetical sorting of the vertces. If multiple verteces is from the same vertex, they will be
+visited in alphabetical order.
+
+3. The ImageStream has not supported pulling using tags yet. It has started working on this feature.
+
+### Risks and Mitigations
+
+Pulling images from mirror registries without the digest specifications could lead to returning different image version from different registry if the image tag mapping is out of sync. But the OpenShift core required image using digests to avoid different versions won't consume this feature at all, so it is not exposed to the risks that anyone who actually uses the feature will be exposed to.
+
+## Design Details
+
+### Test Plan
+
+Update the tests that are currently in the MCO to verify that registries.conf
+requires digest specification for mirrors configured through ImageSourceDigestPolicy.
+
+Update the tests that are currently in the MCO to verify that registries.conf does not
+require digest specification for mirrors configured through ImageSourceTagPolicy.
+
+### Graduation Criteria
+
+GA in openshift will be looked at to
+determine graduation.
+
+#### Dev Preview -> Tech Preview
+
+- Ability to utilize the enhancement end to end
+- End user documentation, relative API stability
+- Sufficient test coverage
+- Gather feedback from users rather than just developers
+- Enumerate service level indicators (SLIs), expose SLIs as metrics
+- Write symptoms-based alerts for the component(s)
+
+#### Tech Preview -> GA
+
+- More testing (upgrade, downgrade, scale)
+- Sufficient time for feedback
+- Available by default
+- Backhaul SLI telemetry
+- Document SLOs for the component
+- Conduct load testing
+
+#### Removing a deprecated feature
+
+- A Jira card [OCPNODE-717](https://issues.redhat.com/browse/OCPNODE-717) was created to record the upgrade of ImageSourceDigestPolicy. The [repositories](https://docs.google.com/document/d/11FJPpIYAQLj5EcYiJtbi_bNkAcJa2hCLV63WvoDsrcQ/edit) currently rely on operator/v1alpha1 ImageContentSourcePolicy(ICSP) will be migrated to config/v1 ImageSourceDigestPolicy.
+
+- The ImageContentSourcePolicy(ICSP) will follow the rules from [Deprecating an entire component](https://docs.openshift.com/container-platform/4.8/rest_api/understanding-api-support-tiers.html#deprecating-entire-component_understanding-api-tiers). The duration is
+12 months or 3 releases from the announcement of deprecation, whichever is longer.  
+
+### Upgrade / Downgrade Strategy
+
+#### Upgrade Strategy
+
+An existing cluster is required to make an upgrade to 4.10 in order to make use of the allow mirror by tags feature. Before the deprecation of ImageContentSourcePolicy, MCO needs to watch for both ImageContentSourcePolicy, and ImageSourceDigestPolicy and create objects.
```

Comment:
is 4.10 correct since this enhancement isn't yet approved?

Reply:
no, it changed from 4.9 to 4.10, now it's 4.11